transaction/src/model/v1/instruction.rs

```diff
@@ -5,6 +5,123 @@ use radix_engine_interface::math::Decimal;
 use radix_engine_interface::types::*;
 use radix_engine_interface::*;
 
+/*
+=================================================================================
+NOTE: For now, we only support "dynamic" addresses for CALL instructions.
+=================================================================================
+This is to reduce the scope of change and make manifest easier to reason about.
```

Comment:
Ahh interesting.

I don't mind this too much, particularly if it's a natural extension of making Address and NamedAddress the same value kind.

But don't feel too strongly.